src/app/nation/nation.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit } from '@angular/core';
+import { Location } from '@angular/common';
+import { Router } from '@angular/router';
+import { CouchService } from '../shared/couchdb.service';
+
+@Component({
+  selector: 'app-nation',
+  templateUrl: './nation.component.html',
+  styleUrls: ['./nation.component.scss']
+})
+export class NationComponent implements OnInit {
+message = '';
+  constructor(
+  	private couchService: CouchService,
```

Comment:
These should be indented to 4, not 3 spaces

Reply:
@paulbert  :+1: